cmd/renterd/main.go

```diff
@@ -392,6 +388,20 @@ func main() {
 	parseEnvVar("RENTERD_LOG_DATABASE_IGNORE_RECORD_NOT_FOUND_ERROR", &cfg.Log.Database.IgnoreRecordNotFoundError)
 	parseEnvVar("RENTERD_LOG_DATABASE_SLOW_THRESHOLD", &cfg.Log.Database.SlowThreshold)
 
+	// deprecated: parse remotes
```

Comment:
Why is this deprecated? What's the successor to these vars?

Reply:
Had to dig a little but this is actually just some slight confusion caused by https://github.com/SiaFoundation/renterd/commit/bf930dffef57d31c5af0f65e77cdbe902dc1f10d where we rename the var to `depWorkerRemoteAddrStr` so I assumed we deprecated it. But we haven't deprecated those.